src/NLog/LayoutRenderers/ExceptionLayoutRenderer.cs

```diff
@@ -66,8 +68,41 @@ public class ExceptionLayoutRenderer : LayoutRenderer, IRawValue
                                                                                                         {"STACKTRACE", ExceptionRenderingFormat.StackTrace},
                                                                                                         {"DATA",ExceptionRenderingFormat.Data},
                                                                                                         {"@",ExceptionRenderingFormat.Serialize},
+                                                                                                        {"HRESULT",ExceptionRenderingFormat.HResult},
+                                                                                                        {"PROPERTIES",ExceptionRenderingFormat.Properties},
                                                                                                     };
 
+        private static readonly HashSet<string> ExcludeDefaultProperties = new HashSet<string>(new[] {
+            "Type",
+            nameof(Exception.Data),
+#if !SILVERLIGHT
+            nameof(Exception.HelpLink),
+#else
+            "HelpLink",
```

Comment:
While I like `nameof`s, I think just "HelpLink" is more clear? The code is more complicated with all the `#if` (and more difficult to refactor, instead of easier to refactor with `nameof`)

Reply:
same for below.

Please remove the `#if`s

Reply:
Removed the `#if`